esp32-pv19.yaml

```diff
@@ -0,0 +1,637 @@
+substitutions:
+  device_name: "Your_device_name"
+  name: "Your_device_name"
+  device_location: "Your_device_location"
+  device_description: "Monitor MUST inverter via RS485"
+  tx_pin: GPIO19
+  rx_pin: GPIO18
+
+esphome:
+  name: ${name}
+  min_version: 2024.6.0
+  name_add_mac_suffix: false
+  comment: ${device_location}
+  project:
+    name: esphome.web
+    version: dev
+
+esp32:
+  board: esp32dev
+  framework:
+    type: arduino
+
+logger:
+
+improv_serial:
+
+wifi:
+  networks:
+    - ssid: !secret wifi_ssid
+      password: !secret wifi_password
+  ap:
+    ssid: ${device_name}
+    password: !secret ap_password
+
+captive_portal:
+
+dashboard_import:
+  package_import_url: github://esphome/example-configs/esphome-web/esp32.yaml@main
```

Comment:
Is it really necessary? I tried to find this repository, but it seems like it was deleted.

Reply:
ESP32 module was initialised by ESPHome Web.
These strings were added to config by initialisation procedure.
I have no idea if these lines are needed for correct import into the dashboard.  Perhaps they are not needed, but the module is already in operation and I do not have another one available for the experiment.
In any case, these lines do not interfere with normal operation